src/Sdk/Sdk.csproj

```diff
@@ -10,7 +10,7 @@
         <DefineConstants>TRACE</DefineConstants>
         <LangVersion>7.3</LangVersion>
         <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-        <TieredCompilationQuickJit>true</TieredCompilationQuickJit>
+        <EnableTrimAnalyzer>false</EnableTrimAnalyzer>
```

Comment:
Trim analyzer is probably not relevant (yet) since you do not compile projects into a single file. And generally disabling it is dangerous

Reply:
this is wrong... I forgot to remove this after I tested something else...